quarkus/runtime/pom.xml

```diff
@@ -109,42 +108,18 @@
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-services</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
@pedroigor just to be sure - do we want to get rid of all star exclusions? I forgot to clarify this one on the Friday sync.

Reply:
Yeah, once the versions from transitive dependencies from Quarkus are the same we don't need to exclude them anymore.

Reply:
I asked because I think I had overdone it a little bit with the removals initially. I added a commit that revert some of the changes. Now it should be in the correct state from my perspective. Please, re-check it, just to be completely sure.

Thank you for your time! (cc: @vmuzikar)

Reply:
Please see https://github.com/keycloak/keycloak/pull/15758#pullrequestreview-1208701082